### ISSUE TITLE
Fronius: PV power limiter not working with some models

### ISSUE THREAD

**Issue by izak** [victronenergy/venus#711]
In Venus v2.60~2 / hub4control v1.2.11, a change was made so that hub4control tracks the PV-inverter state better. This was to fix a problem where too much power is imported from the grid at night, because PV-inverters are incorrectly considered to be in a running state.

(The grid setpoint is increased slightly to give the PV-inverter space to ramp up, but only if it is running).

Turns out, some PV-inverters report an error state even through they are running at full power.

Also, while some Fronius models go into Standby state at night, others cycle through startup/error states during the night.

In other words, the error state sometimes means the PV-inverter is off, and sometimes it doesn't mean that.

For PV-inverters that report an error while running at full power, this means the limiter doesn't work.

Challenge: Find a way to distinguish between a running and non-running PV inverter in error state.

Possible solution: If it makes non-zero power, it's running.
